lib/remap-lang/src/lib.rs

```diff
@@ -240,6 +240,19 @@ mod tests {
             (r#"false || null"#, Ok(()), Ok(().into())),
             (r#"null || "foo""#, Ok(()), Ok("foo".into())),
             (r#". = "bar""#, Ok(()), Ok("bar".into())),
+            (r#"["foo", "bar", "baz"]"#, Ok(()), Ok(vec!["foo", "bar", "baz"].into())),
+            (
+                r#"
+                    .foo = [
+                        "foo",
+                        5,
+                        ["bar"],
+                    ]
+                    .foo
+                "#,
+                Ok(()),
+                Ok(vec!["foo".into(), 5.into(), Value::Array(vec!["bar".into()])].into()),
```

Comment:
I added an `array!` macro that saves you from typing `.into()` for every element. Would be useful here..

https://github.com/timberio/vector/blob/master/src/test_util/mod.rs#L542

Reply:
Sweet 👌